src/main/java/org/broadinstitute/hellbender/tools/spark/sv/RunSGAViaProcessBuilderOnSpark.java

```diff
@@ -221,47 +229,81 @@ static SGAAssemblyResult runSGAModulesInSerial(final String sgaPath,
         // collect runtime information along the way, if user requests
         final List<SGAModule.RuntimeInfo> runtimeInfo = new ArrayList<>();
 
-        String preppedFileName = runAndStopEarly("preprocess", rawFASTQFile, sga, tempWorkingDir, indexer, indexerArgs, runtimeInfo, enableSTDIOCapture);
-        if( null == preppedFileName ){
-            return new SGAAssemblyResult(null, runtimeInfo);
+        final List<String> moduleArgs = new ArrayList<>(20); // 2*10 is not a magic number, just that no SGA modules have more than 10 sensibly tunable parameters
```

Comment:
Why not make this a `Map<String,String>`? I guess you'd have to special case boolean values (ie. if something is just a flag have it use the value 'true' and not write that, but it seems like it would make these a little easier to read.